### ISSUE TITLE
[Extension] Don't show full data:text/html content on generated HTML report page

### ISSUE THREAD

**Issue by tdjsnelling** [IBMa/equal-access#321]
### Summary

When passing HTML into `getCompliance`, the full `data:text/html...` content gets displayed on the generated report page.

This pushes the actual results table waaaay down the page — see screenshot (note the scrollbar):

![Screenshot 2021-02-24 at 17 36 41](https://user-images.githubusercontent.com/6264509/109041902-7517de00-76c7-11eb-9185-15b50cf9bee9.png)

My suggestion would be to instead show just the label value passed to `getCompliance` as these are required and unique. Or if the full HTML content must be shown, moving it below the table of results.

**Comment by tombrunet** [comment]
@tdjsnelling I agree. We need a general solution, but which tool are you using? Browser, Node, Cypress, Karma? For the browser, we don't have a label, but we should at least be able to snip the length somehow.

**Comment by tdjsnelling** [comment]
> @tdjsnelling I agree. We need a general solution, but which tool are you using? Browser, Node, Cypress, Karma? For the browser, we don't have a label, but we should at least be able to snip the length somehow.

Sorry, this is with Node

**Comment by tombrunet** [comment]
Easier to reproduce by:
* Load: https://altoromutual.com/?xyz=jdjklsfhsdakjlfhjkdsahfkjsdhafkjhdslkjfhdsjkfhkjdsahfkjasdhkfjhdsakjlfhkjlsadhfkjadshfkjhsdklfhkjasdhfkjdsahfkjasdhfkjhdskjfhasdkljfhkjadshfkjdsahfklsdhfkjlhsadkjfhkadsjfhkjadshfkjdsahfkljadshfkjhasdkjfhkjadshfjkdsahfkjlhasdkjlfhkjasdhfkjlsadhfjklahdsklfjhskjdahfjkashdfkjsdahfkjhsdakjfhdjksafhkjadlshfkjladshfkjlhadskjlfhklasjdhfkjldsahfjkashdfjklhdasjkfhsadkjlhfkjsadhfkjdaslhfljadskjlafjkdashfkjdashlf
* Save report and look at the HTML saved report.

**Comment by nam-singh** [comment]
@tombrunet below is the screenshot of adjusted screen, please let me know if this works. After 4 lines I have put ellipsis and copy button
<img width="1728" alt="image" src="https://github.com/user-attachments/assets/03e55d0b-f66d-4482-a231-ed51f53ac3cf" />
<img width="1728" alt="image" src="https://github.com/user-attachments/assets/0f7a0867-0384-47bd-a934-9871409c4977" />
